frontend/src/pages/admin/AdminCSVUploadPage.js

```diff
@@ -21,7 +23,19 @@ class AdminCSVUploadPage extends Component {
     console.log('hello');
   };
 
-  handleSubmit = () => {};
+  handleSubmit = async (event, formState) => {
+    event.preventDefault();
+    const { message, success } = await bookFromCSV(this.state.file);
+    if (success) {
+      this.setState({ success: true });
+    } else {
+      this.setState(state => ({
+        errors: [{ message: message, key: state.numSubmits }],
```

Comment:
Just wondering why errors is an array here

Reply:
Good question - I had it there so we could support if BE returned multiple errors, but if that's not the case then we are free to make it a single object.

Reply:
Ahhh that makes sense, unfortunately backend just returns the first error it encounters while trying to load in the books so we could probably just make it a single error